config/config_test.go

```diff
@@ -4,6 +4,8 @@ import (
 	"os/user"
 	"testing"
 
```

Comment:
May we lint these imports? I know this was an error before you change but since you're here.

Reply:
Sure, it's a nitpick but the imports, the declarations for dependencies for this file, are not ordered properly:

```
import (
	"encoding/json"
	"io/ioutil"
	"log"
	"os"
	"testing"

	"github.com/stretchr/testify/assert"
	"github.com/vsco/dcdr/config"
	"github.com/vsco/dcdr/models"
)
```

Unfortunately `goimports` does not catch this lint error due to the spaces between the groups yet, but work is being done here: https://github.com/golang/go/issues/20818

This is not necessary to merge this PR.

Reply:
ordering of imports, got it (i'll try to noticed it next time around)